src/components/Auth/UserFormDialog/controls/SignInMethod.tsx

```diff
@@ -0,0 +1,64 @@
+import { ListDivider } from '@rmwc/list';
+import { Typography } from '@rmwc/typography';
+import React, { useEffect } from 'react';
+import { FormContextValues } from 'react-hook-form/dist/contextTypes';
+
+import { AddAuthUserPayload } from '../../types';
+import styles from './controls.module.scss';
+import { EmailPassword } from './EmailPassword';
+import { PhoneControl } from './PhoneControl';
+
+const ERROR_CODE = 'atLeastOneMethodRequired';
```

Comment:
nit: consider renaming to `ERROR_AT_LEAST_ONE_METHOD_REQUIRED`. Pretty long, but gets the meaning through much more easily.